### PR TITLE
Drop EL7 testing from nightly

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

**Comment by ehelms** [comment]
@evgeni @upadhyeammit @pcreech This is one of the biggest "turn off EL7" knobs and so I want to make sure y'all see it and if y'all are ready to proceed.

**Comment by ekohl** [comment]
Once we merge this, we should clean https://yum.theforeman.org/releases/nightly/el7/ as well so we're not accidentally testing with old packages.

**Comment by ehelms** [comment]
> Once we merge this, we should clean https://yum.theforeman.org/releases/nightly/el7/ as well so we're not accidentally testing with old packages.
> 
> Edit: and https://yum.theforeman.org/katello/nightly/candlepin/el7/ & https://yum.theforeman.org/katello/nightly/katello/el7/ of course.

If we clean those up as soon as merging this, all puppet PRs that require repositories will break IIRC? So are we blocked on merging this till the Puppet changes or can we merge this and then clean those repositories up after the Puppet changes?

**Pull request merged**
